src/browser/screen.js

```diff
@@ -12,6 +12,22 @@ function ScreenAdapter(options, screen_fill_buffer)
 
     console.assert(screen_container, "options.container must be provided");
 
+    const MODE_TEXT = 0;
+    const MODE_GRAPHICAL = 1;
+    const MODE_GRAPHICAL_TEXT = 2;
+
+    const CHARACTER_INDEX = 0;
+    const FLAGS_INDEX = 1;
+    const BG_COLOR_INDEX = 2;
+    const FG_COLOR_INDEX = 3;
+    const TEXT_BUF_COMPONENT_SIZE = 4;
+
+    const FLAG_BLINKING = 0x01;
+    const FLAG_FONT_PAGE_B = 0x02;
+
+    this.FLAG_BLINKING = FLAG_BLINKING;
+    this.FLAG_FONT_PAGE_B = FLAG_FONT_PAGE_B;
```

Comment:
I think these are better as global constants. If those cause a problem in the future, I'll consider using es6 imports.

Reply:
Should I move the two consts to const.js? Are their names ok for a global scope?

Reply:
No, leave them in this file.

Reply:
Ok, I left this unchanged.